zubhub_backend/zubhub/zubhub/urls.py

```diff
@@ -66,6 +66,7 @@
             path('api/help/', HelpAPIView.as_view()),
             path('api/privacy/', PrivacyAPIView.as_view()),
             path('api/faqs/', FAQAPIView.as_view()),
+            path('api/faqs/', ThemeAPIView.as_view()),
```

Comment:
Let's choose a new URL for this

Reply:
Hi @Ritsyy  I was thinking to name it *api/theme*. What do you think ?